utils/io.go

```diff
@@ -107,39 +107,60 @@ func FormatDecimal(integer int, exp int) string {
 	return string(bytes)
 }
 
-// Return env variable value for the given key,
-// or the fallback value if undefined or empty
+// Returns envvar value or fallback if undefined or empty.
 func GetEnv(key, fallback string) string {
-	if value, ok := os.LookupEnv(key); ok {
-		if len(value) > 0 {
-			return value
-		}
+	if value, ok := os.LookupEnv(key); ok && len(value) > 0 {
+		return value
 	}
```

Comment:
You can simplify by using os.Getenv . It returns the value or empty (if empty or non-existing).